test/parallel_api/numeric/numeric.ops/scan_2.pass.cpp

```diff
@@ -145,10 +145,9 @@ DEFINE_TEST_1(test_scan_non_inplace, TestingAlgoritm)
 
     // specialization for hetero policy
     template <typename Policy, typename Iterator1, typename Iterator2, typename Size>
-    typename ::std::enable_if<
-        oneapi::dpl::__internal::__is_hetero_execution_policy<typename ::std::decay<Policy>::type>::value &&
-            is_base_of_iterator_category<::std::random_access_iterator_tag, Iterator1>::value,
-        void>::type
+    ::std::enable_if_t<
+        oneapi::dpl::__internal::__is_hetero_execution_policy<::std::decay_t<Policy>>::value &&
```

Comment:
Potentially redundant `std::decay_t`

Reply:
From point of view how ```__is_hetero_execution_policy``` is implemented at this moment - ```std::decay_t``` is required here.